### DISCUSSION TITLE
About connecting controller to UGS

### DISCUSSION THREAD

**Discussion by Yashk09** [winder/Universal-G-Code-Sender#2329] [Help]
I have arduino uno R3 board which I tried to connect with UGS but UGS have an error "the setup wizard it tells me "this set up wizard does not know how to configure this type of controller." 
So please help me to solve this problem
I have Grbl v0.9j and UGS 2.0.20

**Comment by breiler** [comment]
You will need to install the latest nightly build from here:
https://github.com/winder/Universal-G-Code-Sender#downloads

Previously reported here: #2319
